src/styles/components/_home-search-box.scss

```
@@ -1,18 +1,65 @@
 @import '../abstracts/_abstracts-dir';
 
 .home-search-box {
-  @include flex-center();
+  display: flex;
+  width: 80%;
 
-  &__button {
-    @include button-darken-effect($picton-blue);
+  background-color: $light;
+  box-shadow: rem(5px 10px 20px) 0px box-shadow-opacity(0.16);
```

css `box` should come before `color`
```suggestion
  box-shadow: rem(5px 10px 20px) 0 box-shadow-opacity(0.16);
  background-color: $light; 
```

---

I agree, but the color would be separated like this:
```
  display: flex;
  width: 80%;
  box-shadow: rem(5px 10px 20px) 0 box-shadow-opacity(0.16);

  background-color: $light;
```